src/psen_scan_driver.cpp

```diff
@@ -91,6 +92,12 @@ int main(int argc, char** argv)
         .scanRange(scan_range)
         .enableDiagnostics();
 
+    if (getOptionalParamFromServer<bool>(pnh, PARAM_FRAGMENTED_SCANS, configuration::DEFAULT_FRAGMENTED_SCANS))
+    {
+      ROS_INFO("Using fragmented scans.");
+      config_builder.enableFragmentedScans();
+    }
+
```

Comment:
Conflicts with ac95d01a6f67f7c8bc9250065b21ec1acdb5b29f.

Reply:
If `enableFragmentedScans()` gets passed a value it should be possible.